templates/prelogin/home.html

```diff
@@ -5,14 +5,14 @@
 {% block js-inline %}{{ block.super }}
     <script>
         $(function(){
-            var signUpForm = $('form.form-signup');
```

Comment:
this was returning two forms, and submitting them was choosing the `e=` param from the second one.